app/theme.tsx

```diff
@@ -61,7 +61,7 @@ const lightPalette = {
   primary: {
     main: "#FFBB6B",
     light: "#FFD580",
-    dark: "#B3773E",
+    dark: "#000000",
```

Comment:
These changes seems wrong. Also where are you using them as the only place they are going to have any effect are the buttons and those also have custom colors? 

I just checked, and they are affecting the sorting buttons on the blog page in light mode. Please fix that and check other areas as well.